src/signals/incident/components/IncidentForm/index.js

```diff
@@ -29,7 +30,9 @@ class IncidentForm extends React.Component {
 
   setForm(form, incidentContainer) {
     this.form = form;
+    const incident = get(incidentContainer, 'incident', {});
```

Comment:
Why do you need an extra dependency here? Wouldn't 
```jsx
const { incident = {} } = incidentContainer || {};
```
also do the trick?

Reply:
Sure